ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/prefix_lists.schema.yml

```diff
@@ -27,13 +25,11 @@ keys:
             keys:
               sequence:
                 type: int
-                required: true
```

Comment:
I don't think we should change the required fields for each list item, so I would revert the changes concerning `sequence` and `action`

Reply:
Fixed